examples/interactive-containers/interactive-3d-slicer/custom_startup.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+# The content of /output is soft links to files in /input, hence copy will be skipped - so we delete the output dir first
+rm -r /output/*
+# Now we can copy the files
+cp -r /input/* /output/
```

Comment:
I think we should only copy the dicom_data and file_data subdirectories, so that we don't copy the dataset.csv.